nvflare/edge/emulator/cifar10_cpp.json

```diff
@@ -0,0 +1,4 @@
+{
+    "class_path": "et_cpp_task_processor.ETCppTaskProcessor",
+    "class_args": {"et_binary_path": "train_cifar10", "et_model_path": "cifar10.pte", "data_path": "/Users/yuantingh/Downloads/cifar-10-batches-bin/data_batch_1.bin"}
```

Comment:
same here

Reply:
Should change this to the new JSON format.